datafusion/physical-expr/src/functions.rs

```diff
@@ -2628,6 +2530,87 @@ mod tests {
             Int32Array
         );
         #[cfg(feature = "unicode_expressions")]
+        test_function!(
```

Comment:
Thank you @SteveLauC 

Any chance you would be willing to port these tests to sqllogictests (as a follow on PR)? They can probably be implemented easily in https://github.com/apache/arrow-datafusion/blob/83d15e8b223e1af2c553ec54afa5c1ffac7c340e/datafusion/sqllogictest/test_files/scalar.slt#L4

Reply:
Thanks for your review!

> Any chance you would be willing to port these tests to sqllogictests (as a follow on PR)? 

Sure, I wil do it in another PR:)